tap-agent/src/agent/sender_allocation.rs

```diff
@@ -565,7 +615,10 @@ impl SenderAllocationState {
         }
     }
 
-    async fn store_invalid_receipts(&self, receipts: &[ReceiptWithState<Failed>]) -> Result<()> {
+    async fn store_invalid_receipts(
+        &mut self,
```

Comment:
After the last commit it doesn't need to `mut` anymore
```suggestion
        &self,
```

Reply:
It does because we are updating the self.invalid_receipt_fees.value which is mut.

Reply:
> It does because we are updating the self.invalid_receipt_fees.value which is mut.
Ohh yeah you're right. Though the thing is that `checked_add` does not add in place. That's why right now you don't need mut (I just tested locally).